scripts/dotnet.py

```diff
@@ -555,7 +555,7 @@ def get_dotnet_path() -> str:
     return dotnet_path
 
 
-def get_dotnet_version(
+def get_dotnet_version_path(
```

Comment:
looking at this method, it doesn't seem to return a path, it seems to return a version number. Is this the right name?

Reply:
Good point. I think based on how it is used it would make sense, but I switched the name to match what it does. I updated the name to get_dotnet_version_from_path as it is getting the dotnet version based on the folder name in the sdk_path. I am definitely open to suggestions for a better name.

Reply:
`get_dotnet_version_from_path` is perfect, I misunderstood it before as thinking it meant getting a path to a dotnet version